abci/client/client.go

```diff
@@ -37,8 +37,16 @@ type Client interface {
 	// with the exception of `CheckTxAsync` which we maintain
 	// for the v0 mempool. We should explore refactoring the
 	// mempool to remove this vestige behavior.
-	SetResponseCallback(Callback)
-	CheckTxAsync(context.Context, *types.RequestCheckTx) (*ReqRes, error)
+	SetGlobalCallback(GlobalCallback)
+	GetGlobalCallback() GlobalCallback
```

Comment:
@dudong2 
```suggestion
	SetResponseCallback(Callback)
	GetResponseCallback() Callback
```

Unless it’s absolutely necessary to explicitly name it `GlobalCallback`, it would be better to align the naming with Comet as much as possible.

Reply:
ditto